guides/get-started-microservices-on-kubernetes/simple-microservice-part2.adoc

```diff
@@ -0,0 +1,134 @@
+= {simple-microservice-part2}
+:summary: ONELINE EXPLANATION OF THE GUIDE
+:includedir: ../_includes
+include::{includedir}/_attributes.adoc[]
+include::./_includes/_titles.adoc[]
+// you can override any attributes eg to lengthen the
+// time to complete the guide
+:prerequisites-time: 10
+
+In this guide, you will learn how-to run the Docker Image we built in link:simple-microservice-part1[{simple-microservice-part1}] on Kubernetes.
+
+include::{includedir}/_prerequisites.adoc[]
+* You have completed link:simple-microservice-part1.adoc[Create a JAX-RS application using Bootable Jar - part one: bare metal]
+
+include::_includes/_constants.adoc[]
+
+== Pushing the Docker Image to an Image Registry
+
+To make the `{docker-image-name}:latest` Docker Image available to Kubernetes, we need to push it to some Image Registry that is accessible by the Kubernetes cluster we want to use;
+
+There are many options to achieve this; in this guide, we will push the `{docker-image-name}:latest` Docker Image, to the link:quay.io[quay.io] Image Registry;
+
+=== Quay.io
+
+You can skip this step in case you prefer to re-use the `quay.io/{quay-io-account-name}/{docker-image-name}` Docker Image, that we already pushed to link:quay.io[quay.io] in the `{quay-io-account-name}` account;
```

Comment:
I think we should not mention here personal accounts. Better will be to use something general.

Reply:
+1 Maybe @jmesnil can comment about this?

Reply:
@yersan , @jmesnil , @mnovak1 that would be nice to use some account other than mine: please can you suggest one and, also, provide the necessary credentials to me so that I can add these images there?

Reply:
@tommaso-borgato not from my side, I guess you need to get in touch primarily with Jeff